apim/3.x/README.adoc

```diff
@@ -1,78 +1,97 @@
 [[graviteeio-api-management-helm-chart]]
 == Gravitee.io API Management Helm Chart
 
-*Chart supported versions: 3.0.x and higher*
+*Chart supported versions: 3.0.x and higher*. 
 
 === Components
 
 This chart will deploy the following:
 
-* Gravitee Management API
-* Gravitee Management UI
-* Gravitee Portal UI
-* Gravitee Gateway
-* MongoDB replica-set (optional dependency)
-* Elasticsearch cluster (optional dependency)
+* Gravitee Management API.
+* Gravitee Management UI.
+* Gravitee Portal UI.
+* Gravitee Gateway.
+* MongoDB replica set  or PostgresSQL (optional dependency).
+* Elasticsearch Cluster (optional dependency).
 
-=== Installing
+=== 1. Install The Gravitee Helm Chart Repo
 
-* Add the Gravitee.io helm charts repo
+* Add the official Gravitee.io Helm chart repo: 
 +
 ....
 $ helm repo add graviteeio https://helm.gravitee.io
 ....
-* Install it
-+
-....
-$ helm install --name graviteeio-apim3x graviteeio/apim3
-....
 
-=== Create a chart archive
 
-To package this chart directory into a chart archive, run:
+=== 2. Create The Configuration File & Set Custom Values 
+Before you continue installing the Helm chart, create a `values.yml` configuration file and set any applicable custom values in it. 
 
-....
-$ helm package .
-....
-
-=== Installing the Chart
-
-To install the chart from the Helm repository with the release name
-`+graviteeio-apim3x+`:
 
-[source,bash]
 ----
-$ helm install --name graviteeio-apim3x graviteeio/apim3
+# Management console version
+ui:
+  image:
+    tag: 3.10.0
+
+# Gateway version
+gateway:
+  image:
+    tag: 3.10.0
+    
+# Portal version
+portal:
+  image:
+    tag: 3.10.0
+
+# Management API version
+api:
+  image:
+    tag: 3.10.0
+
+# gateway proxy configuration
+# (remove '/gateway' prefix from URL before sending request)
+gateway:
+  ingress:
+    path: /gateway/(.*)
+    annotations:
+      nginx.ingress.kubernetes.io/rewrite-target: /$1
+
+# ES endpoint
+es:
+  endpoints:
+    - http://elastic-elasticsearch-data.default.svc.cluster.local:9200
 ----
 
-*Note:* If you're using Helm 3, the name parameter is no more valid.
-Please check
-https://helm.sh/docs/faq/#release-names-are-now-scoped-to-the-namespace[https://helm.sh/docs/faq/#release-names-are-now-scoped-to-the-namespace]
 
-To install the chart using the chart archive, run:
+==== PostgresSQL Setup 
+For PostgrestSQL, use the information below in `values.yml` and replace the `username`, `password`, 
+`URL` and `database name` with details for your specific instance. 
 
-....
-$ helm install apim3-3.0.0.tgz
-....
-
-=== Configuration
+----
+jdbc:
```

Comment:
I think it could be confusing that the postgresql part of the configuration file is located here,
But the mongo part of the configuration file is located under "Configure Optional Dependencies" chapter